README.md

```diff
@@ -64,8 +64,14 @@ we test it on 200 patients dataset, got the **AUC** as below:
 |AVG_MAX|No|
 |AVG_MAX_LSE|No|
 
-> We are now implementing heatmap options for `AVG_MAX`, `AVG_MAX_LSE`, and `PROB`, So before that you can use the supported `global_pool` options to train a model and plot the heatmaps.
+* We also provide heatmap comparision here, including AVG, [LSE](https://arxiv.org/abs/1705.02315), and our own PROB pooling.
 
+||&emsp;&emsp;&emsp;&emsp; original &emsp;&emsp;&emsp;&emsp; [AVG (dev mAUC:0.895)](config/example_AVG.json) &emsp; [LSE (dev mAUC:0.896)](config/example_LSE.json) &emsp; [PROB (dev mAUC:0.896)](config/example_PROB.json)|
+|------|-----|
```

Comment:
The header here is misplaced when it is displayed, and the table needs to be merged and centered. The Markdown syntax may not be supported, so it is recommended to use HTMAL. Here I have simply written one, you can click edit and copy to use it
`<table>
  <tr>
    <td> </td>
		<td>original</td>
		<td>AVG (dev mAUC:0.895)</td>
    <td>LSE (dev mAUC:0.896)</td>
    <td> PROB (dev mAUC:0.896)</td>
	<tr>
  <tr>
    <td>Cardiomegaly </td>
		<td colspan="4"><img src="Cardi.jpg"></td>
	<tr>
  <tr>
    <td>Atelectasis </td>
		<td colspan="4"><img src="Cardi.jpg"></td>
        <tr>
    <td>Cardiomegaly </td>
		<td colspan="4"><img src="AT.jpg"></td>
           <tr>
    <td>Consolidation </td>
		<td colspan="4"><img src="PE.jpg"></td>
</table>`

Reply:
Done.